### PR TITLE
Introduce services

### PR DESCRIPTION
**Pull request by ashtonmoomoo**

### CODE REVIEW

File: app/questions.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from sympy import symbols, latex, Eq
+from . import services
```
Comment (odrusso) [inline app/questions.py:2]: Could this be like, maths_service.py?
Then there are other services, db_service, hamburger_service, etc

Comment (ashtonmoomoo) [reply app/questions.py:2]: good call

File: app/services.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from time import time
+import random
+
+def integer_coefficients(amount=1, number_range=[1, 10], seed=time()):
+
+    random.seed(a=seed)
```
Comment (odrusso) [inline app/services.py:6]: I assume passing the seed in here means that the .choice and .randint will always return the same thing?

Comment (ashtonmoomoo) [reply app/services.py]: correct

**Review by odrusso — APPROVED** [review]
good stuff

**Pull request merged**